web/src/app/core/services/authentication.service.ts

```diff
@@ -9,87 +9,75 @@ import * as firebase from 'firebase';
 import { auth, User } from 'firebase/app';
 
 // Rxjs operators
-import { from, of, Observable } from 'rxjs';
-import { concatMap, filter, first, switchMap, takeUntil, tap } from 'rxjs/operators';
+import { from, of, Observable, Subscription } from 'rxjs';
+import { concatMap, filter, first, switchMap, tap } from 'rxjs/operators';
 
 // Third party modules
 import { DeviceDetectorService } from 'ngx-device-detector';
 
 // Dashboard hub models
 import { LoginAuditModel, ProfileModel } from '../../shared/models/index.model';
-import { SpinnerService } from './spinner.service';
+import { ActivityService } from './activity.service';
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthenticationService {
 
+  private subscriptions: Subscription[] = [];
```

Comment:
We are following the alphabetical ordering for variable names

Reply:
Private needs to come first according to the linter, but I will order within the public correctly

Reply:
correct :)